src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -333,8 +333,10 @@
   "entity.worker.inventoryfullchestfull": "Please make space in my chests/racks. I am not able to dump my inventory.",
 
   "entity.builder.messagebuildstart": " started building %s.",
-  "entity.builder.messagebuildcomplete": " finished building %s.",
+  "entity.builder.messagebuildcomplete": "finished building %s.",
+  "entity.builder.messagebuildcomplete_manual": "finished building %s. You can asign them another task now!",
   "entity.builder.messagebuildcomplete_generic": "%s finished building %s.",
+  "entity.builder.messagebuildcomplete_generic_manual": "Builder %s finished building %s. You can asign them another task now!",
```

Comment:
```suggestion
  "entity.builder.messagebuildcomplete_generic_manual": "%s finished building %s. You can assign them another task now!",
```

Reply:
I did this one first, before i noticed that i can't do that with the other one, seems like i forgot to remove the "Builder" :D